scripts/models/MemberOfCongress.js

```diff
@@ -0,0 +1,39 @@
+(function(module) {
+  function MoC(opts) {
+    for (keys in opts) {
+      this[keys] = opts[keys];
+    }
+    if (this.type === 'sen') {
+      this.repOf = 'Sen. ' + this.state;
+      this.displayType = 'Senate';
+      this.subtitle = 'Senator';
+    } else {
+      this.repOf = this.state + '-' + this.zeroPadding();
+      this.displayType = 'House';
+      this.subtitle = this.state + '-' + this.district;
+    }
+  }
+
+  MoC.prototype.zeroPadding = function() {
+    var zeros = '00';
+    district = this.district.toString();
+    return zeros.substring(0, zeros.length - district.length) + district;
+  };
+
+  MoC.all = firebasedb.ref('mocData/').once('value').then(function(snapshot) {
+    var MoCs = snapshot.val();
+    // Do we need to check if party exists here? It was in the missing members code, but I can't see any examples of MoCs without party
+    return Object.keys(MoCs).map(function(key) {
```

Comment:
I'm not sure, I think it might have been a hold over from when our data was less complete. What should be checked here is 'in_office === true'